README.md

```diff
@@ -1,3 +1,17 @@
+# Execução
+
+[Será necessário que possua o Docker instalado em seu sistema operacional para executar esse projeto.](https://docs.docker.com/get-docker/)
+
+Após a instalação:
+
+`docker compose build`
+
+`docker network create reverse-proxy`
```

Comment:
Maybe we can remove this network and only add this at production extending the default docker-compose file. What you think about this suggestion?